packages/zambdas/src/shared/pdf/discharge-summary-pdf.ts

```diff
@@ -0,0 +1,715 @@
+    ];
+
+    const leftX = pdfClient.getLeftBound();
+    const colGap = 5;
+    const colWidth = (pdfClient.getRightBound() - leftX - colGap) / 2;
+    const rightX = leftX + colWidth + colGap;
+
+    let y = pdfClient.getY();
+
+    const lineHeight = textStyles.regular.font.heightAtSize(textStyles.regular.fontSize);
+    const rowSpacing = 6;
+
+    vitals.forEach(([label1, value1, label2, value2]) => {
+      pdfClient.drawTextSequential(
+        `${label1}: `,
+        {
+          ...textStyles.bold,
+          newLineAfter: false,
+        },
+        {
+          leftBound: leftX,
+          rightBound: leftX + colWidth,
+        }
+      );
+
+      const label1Width = pdfClient.getTextDimensions(`${label1}: `, textStyles.bold).width;
+      pdfClient.drawTextSequential(
+        `${value1}`,
+        {
+          ...textStyles.regular,
+          newLineAfter: true,
+        },
+        {
+          leftBound: leftX + label1Width,
+          rightBound: leftX + label1Width + colWidth,
+        }
+      );
+
+      pdfClient.setY(y);
+      pdfClient.drawTextSequential(
+        `${label2}: `,
+        {
+          ...textStyles.bold,
+          newLineAfter: false,
+        },
+        {
+          leftBound: rightX,
+          rightBound: rightX + colWidth,
+        }
+      );
+
+      const label2Width = pdfClient.getTextDimensions(`${label2}: `, textStyles.bold).width;
+      pdfClient.drawTextSequential(
+        `${value2}`,
+        {
+          ...textStyles.regular,
+          newLineAfter: true,
+        },
+        {
+          leftBound: rightX + label2Width,
+          rightBound: rightX + label2Width + colWidth,
+        }
+      );
+
+      y -= lineHeight + rowSpacing;
+      pdfClient.setY(y);
+    });
+    pdfClient.drawSeparatedLine(separatedLineStyle);
+  };
+
+  const drawAllergies = (): void => {
+    pdfClient.drawText('Known Allergies', textStyles.subHeader);
+    const allergies = data.allergies?.join('; ');
+    const notes = data.allergiesNotes?.length ? '; ' + data.allergiesNotes.join('; ') : '';
+    const fullLine = allergies + notes;
+    pdfClient.drawText(fullLine, textStyles.regular);
+    pdfClient.drawSeparatedLine(separatedLineStyle);
+  };
+
+  const drawCurrentMedications = (): void => {
+    pdfClient.drawText('Current Medications', textStyles.subHeader);
+    const medications = data.currentMedications?.join('; ');
+    const notes = data.currentMedicationsNotes?.length ? '; ' + data.currentMedicationsNotes.join('; ') : '';
+    const fullLine = medications + notes;
+    pdfClient.drawText(fullLine, textStyles.regular);
+    pdfClient.drawSeparatedLine(separatedLineStyle);
+  };
+
+  const drawInHouseLabs = (): void => {
+    pdfClient.drawText('In-House Labs', textStyles.subHeader);
+    if (data.inHouseLabs?.orders.length) {
+      pdfClient.drawText('Orders:', textStyles.subHeader);
+      data.inHouseLabs?.orders.forEach((lab) => {
```

Comment:
This is a readability nit. A lot of the labs terminology is overloaded and so it becomes hard to follow pretty fast. Your other forEach loops have this issue as well.

lab -> Laboratory
test or testItem -> actual test performed
order -> set of tests (can be singleton)

You probably want order or testItem here
```suggestion
      data.inHouseLabs?.orders.forEach((order) => {
```